frontend/language/src/nb.json

```
@@ -755,6 +755,8 @@
   "resourceadm.deploy_version_upload_button": "Last opp dine endringer",
   "resourceadm.error_back_to_dashboard": "Gå tilbake til dashbord",
   "resourceadm.error_page_text": "Du har nådd en ugyldig adresse",
+  "resourceadm.import_resource_empty_list": "Det finnes ingen servicer i miljøet du valgte",
```

Can we clarify this message even more? 🤔 What about something like this? 
```suggestion
  "resourceadm.import_resource_empty_list": "Det finnes ingen servicer i {{env}}-miljøet",
```

Then we can pass variables into the text-resource like this:

```
 t('resourceadm.import_resource_empty_list', {
    env: myAwesomeVariableInReactComponent
 });
```